packages/office-addin-lint/src/lint.ts

```diff
@@ -15,6 +15,7 @@ const eslintConfigPath = path.resolve(__dirname, "../config/.eslintrc.json");
 const eslintTestConfigPath = path.resolve(__dirname, "../config/.eslintrc.test.json");
 
 function execCommand(command: string) {
+  console.log(`> ${command}`);
```

Comment:
What is this about?

Reply:
This just adds the actual command that was run as part of the output.  It's useful to see the paths that are part of the command . . . but maybe not necessary.

Reply:
I think you should take this out.